src/collection/dimensions.js

```diff
@@ -266,6 +266,33 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, height, paddingObject, relativeTo ) {
+        // Assuming percentage is number from 0 to 1
```

Comment:
If you assume the number is from 0 to 1, where do you convert a users value from '50%' to 0.5?